public/history.php

```diff
@@ -187,6 +189,15 @@ function resize() {
   }
 </script>
 
+<script>
+  function convertDate() {
+    var field = document.gotodateform.dateinput;
+    var timestamp = new Date(field.value).getTime() / 1000;
+    document.gotodateform.d.value = timestamp;
+    return true;
+ }
```

Comment:
nit: Seems there is some missing white space here, I think maybe ESLint is not running on these PHP files 😅

```suggestion
  }
```